service/src/main/java/uk/gov/hmcts/reform/fpl/model/docmosis/DocmosisAnnexDocuments.java

```diff
@@ -30,7 +30,6 @@
     @Deprecated
```

Comment:
all these deprecated fields can be removed. That will remove stuff in  SampleCaseSubmissionTestDataHelper also